src/components/fields/SQFormDropdown/SQFormDropdown.tsx

```diff
@@ -137,7 +137,7 @@ function SQFormDropdown({
           {label}
         </InputLabel>
         <Select
-          sx={classes.selectHeight}
+          sx={(classes.selectHeight, {fontWeight: 600})}
```

Comment:
Can we put the `fontWeight` into the `classes.selectHeight` class?

Reply:
done, thanks @aharpt